app/assets/javascripts/govuk_publishing_components/components/image-card.js

```diff
@@ -0,0 +1,19 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function ImageCard ($module) {
```

Comment:
Nice addition, just wondering if we need to highlight when this gets released that this will need including in the assets for any application using the image card. I'm wondering if a breaking change is appropriate - what's the worst that can happen if this JS weren't included in an application using the image card with a Youtube video?

Reply:
If there's no Javascript included, it will still render the fallback link which will be rendered in the case where JS is disabled/doesn't run or cookies are disabled.

![Screenshot 2023-01-06 at 14 58 10](https://user-images.githubusercontent.com/3727504/211037673-e666ce3c-6c47-4a82-8960-35ba81205f85.png)

Reply:
That's good to know. Will still need to check this gets deployed to applications correctly.